client/src/pages/MarktForm.js

```diff
@@ -57,6 +84,18 @@ export default function MarketForm({ onAddMarket }) {
           onChange={updateMarket}
           value={market.description}
         />
+        <input
+          type="file"
+          multiple
+          onChange={(e) => {
+            setImageUploads(e.target.files);
+          }}
+        ></input>
+        <ImageWrapper>
+          {market.images.map((images, index) => (
+            <Img key={index + images} src={images} />
+          ))}
```

Comment:
The usage of `index` as part of the key is not really necessary since the image URL should be unique already.

```suggestion
          {market.images.map((image, index) => (
            <Img key={index + image} src={image} />
          ))}
```